Cargo.toml

```diff
@@ -31,6 +31,11 @@ phf = "0.7"
 version = "0.6"
 default-features = false
 
+[dependencies.multipart]
+version = "0.2"
```

Comment:
Doing this makes cargo get the latest 0.2.X release. I prefer to do it like this to get those sweet bug fixes while avoiding as much breaking as possible.